report-api/src/api/services/report_service.py

```diff
@@ -65,14 +64,66 @@ def _finalize_pdf(
         """Route to chunk only when statement_report has groupedInvoices; else render directly."""
         is_statement = 'statement_report' in (template_name or '')
         has_grouped_invoices = bool((template_args or {}).get('groupedInvoices'))
+        footer_html = None
         if is_statement and has_grouped_invoices:
             return ChunkReportService.create_chunk_report(
                 template_name,
                 template_args,
                 generate_page_number,
             )
-        return ReportService.generate_pdf(html_out, generate_page_number)
+        return ReportService.generate_pdf(html_out, generate_page_number, footer_html=footer_html)
```

Comment:
Looks like footer_html is none in L67.

The function also defaults to None, so we can just remove footer_html variable here.